docker-compose/run_selenium_tests_remote.sh

```diff
@@ -50,7 +50,7 @@ echo_msg
 # Set bash builtins for safety
 set -e -u -o pipefail
 
-export USE_NEW_PERM_SCREEN=false
```

Comment:
Just to capture the note from eng sync, let's also remove the `-p` option altogether, both in this and the local version in the other file.